easybuild/tools/options.py

```diff
@@ -134,6 +134,7 @@ def basic_options(self):
             'job': ("Submit the build as a job", None, 'store_true', False),
             'logtostdout': ("Redirect main log to stdout", None, 'store_true', False, 'l'),
             'only-blocks': ("Only build listed blocks", None, 'extend', None, 'b', {'metavar': 'BLOCKS'}),
+            'rebuild': ("Rebuild the software without skipping dependency tests", None, 'store_true', False),
```

Comment:
I would change the help message to:

`"Rebuild software, even if module already exists (don't skip OS dependencies checks)"

and maybe revisit the help message for `--force` and clarify it if needed?